clients/web/test/spec/widgetsSpec.js

```diff
@@ -52,6 +54,35 @@ describe('Test widgets that are not covered elsewhere', function () {
         }, 'progress to report an error');
 
         runs(function () {
+            var origOnMessage = girder.eventStream._eventSource.onmessage;
+            girder.eventStream._eventSource.onmessage = function (e) {
+                try {
+                    origOnMessage(e);
+                } catch (err) {
+                    onMessageError += 1;
+                }
+            };
+            var stream = girder.events._events['g:navigateTo'][0].ctx.progressListView.eventStream;
+            stream.on('g:error', function () { errorCalled += 1; });
+            stream.on('g:event.progress', function () {
+                throw 'intentional error';
+            });
+            _setProgress('success', 0);
+        });
+        waitsFor(function () {
+            return onMessageError == 1;
+        }, 'bad progress callbcak to be tried');
```

Comment:
I'll be the designated spell checker.  :smirk_cat: 

callbcak here and at line 80